aspnetcore/performance/caching/hybrid/includes/overview.md

```diff
@@ -83,4 +83,4 @@ For more information, see the following resources:
 
 * <xref:performance/caching/hybrid>
 * [GitHub issue dotnet/aspnetcore #54647](https://github.com/dotnet/aspnetcore/issues/54647)
```

Comment:
When you see these, we developed a format years ago. We don't all do it, but I think they compose well and make SEO + accessibility better ...

```suggestion
* [Hybrid Cache API proposal (`dotnet/aspnetcore` #54647)](https://github.com/dotnet/aspnetcore/issues/54647)
```